types/react-big-scheduler/index.d.ts

```diff
@@ -15,6 +15,16 @@ export interface SchedulerProps {
     nextClick(schedulerData: SchedulerData): void;
     onSelectDate(schedulerData: SchedulerData, date: string): void;
     onViewChange(schedulerData: SchedulerData, view: View): void;
+    newEvent(
+    schedulerData: any,
+    slotId: any,
+    slotName: any,
+    start: any,
+    end: any,
+    type: any,
+    item: any
```

Comment:
If there's a good reason for these to be `any`, then please let me know. It kind of defeats the purpose of using TS if you just use `any` everywhere. `schedulerData` has a type as evidenced in the proceeding lines, so at the very least, I'd expect that to be properly typed.